<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>635</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>5000</width>
    <height>3000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="CammerWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>541</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Open">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="text">
    <string>打开相机</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Close">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="text">
    <string>关闭相机</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Start">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="text">
    <string>开始抓图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Stop">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="text">
    <string>停止抓图</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>561</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>23</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_Statistic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>571</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Statistics</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_Rote">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>200</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_Gain">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>290</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Width">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>460</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Height">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>460</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Resolution">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="text">
    <string>设置分辨率</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Rote">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>200</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Gain">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>290</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Rote">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>160</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>设置帧率</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Gain">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>250</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>设置增益</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Exposure">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>380</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Exposure">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>340</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>设置曝光</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_Exposure">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>380</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Onestart">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="text">
    <string>单张采集</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>430</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>宽度           *           高度</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CammerWidget</class>
   <extends>QWidget</extends>
   <header location="global">Src/cammerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
